### DISCUSSION TITLE
[Enhancement] Server Password Disable Wildcard

### DISCUSSION THREAD

**Discussion by nickkeane** [mbround18/valheim-docker#176] [Q&A] [answered]
Can a special string like a wildcard be used to disable the password i.e. PASSWORD="*" or PASSWORD="DISABLE"?

I have ValheimPlus set up in the config to disable password requests. According to one of their issue responses, it doesn't work when the password gets populated. In this case, a blank or non-specified password results in '12345' instead of an empty or non-existent config option.

**Comment by mbround18** [comment, marked as answer]
Use longship to set a disabled password
